spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe "Mechanics show spec" do
+  before(:each) do
+    @park1 = AmusementPark.create!(name: "Denver", admission_cost: 33)
+    @ride1 = @park1.rides.create!(name: "Tower", thrill_rating: 8, open: true)
+    @ride2 = @park1.rides.create!(name: "Doom", thrill_rating: 8, open: false)
+    @ride3 = @park1.rides.create!(name: "Merry-go-round", thrill_rating: 3, open: true)
+    @mechanic1 = @ride1.mechanics.create!(name: "Martin", years_experience: 10)
+    @mechanic2 = @ride1.mechanics.create!(name: "Rodrigo", years_experience: 9)
+
+    MechanicRide.create!(ride_id: @ride2.id, mechanic_id: @mechanic1.id)
+  end
+
+  describe '#us 1' do
+    it 'Displays a mechacnics attributes and what rides they are working on' do
+
+      # When I visit a mechanic show page
+      visit "/mechanic/#{@mechanic1.id}"
+      # save_and_open_page
+      # I see their name, years of experience, and the names of all rides they are working on.
+      expect(page).to have_content("Name: Martin")
+      expect(page).to have_content("Experience: 10")
+      expect(page).to have_content("Tower")
+      expect(page).to have_content("Doom")
+
+    end
```

Comment:
Now this is a good test, you see how on line 33 you add the text `"Experience: "` into the test, when dealing with number values, these are the tests we want to see. Or use `within` blocks.